### PR TITLE
feat: add version export

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Like similar PRs in App Kit, we also add a version export 😄

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -18,6 +18,9 @@ const configureRollupPlugins = (options = {}) =>
   [
     replace({
       'process.env.NODE_ENV': JSON.stringify('production'),
+      'process.env.npm_package_version': JSON.stringify(
+        process.env.npm_package_version
+      ),
```
Comment (tdeekens) [inline rollup.config.js]: Nice. If this works I'll simplify app-kit again too again.

Comment (montezume) [reply rollup.config.js]: But we don't use lerna here for bumping versions... I think that's why we can have this simpler

Comment (montezume) [reply rollup.config.js]: Screen Shot 2019-06-06 at 10 51 42 AM

screenshot of cjs file

File: src/index.bundlespec.js [hunk 2]
```diff
@@ -9,6 +9,12 @@ describe('exports', () => {
     expect(customProperties).toBeTruthy();
   });
 
+  describe('version', () => {
+    it('should match package.json version', () => {
+      expect(version).toMatch(process.env.npm_package_version);
```
Comment (montezume) [inline src/index.bundlespec.js]: bundle tests 😄

Comment (montezume) [reply src/index.bundlespec.js]: or should I directly import from package.json 🤔

File: rollup.config.js (hunk 1 above)
Comment (tdeekens) [reply rollup.config.js]: Nice.

File: src/index.bundlespec.js (hunk 2 above)
Comment (tdeekens) [reply src/index.bundlespec.js]: I think either is fine.

**Pull request merged**